pkg/controller/template/render.go

```diff
@@ -257,33 +256,34 @@ func MachineConfigFromIgnConfig(role string, name string, ignCfg *ignv2_2types.C
 	}
 }
 
-func transpileToIgn(files, units []string) (*ignv2_2types.Config, error) {
-	var ctCfg cttypes.Config
+func transpileToIgn(files, units []string) (*igntypes.Config, error) {
```

Comment:
Can we add a top level comment here explaining that the data we're unmarshalling has literal contents for `source`?

Reply:
sure

Reply:
done